src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -410,6 +496,27 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
     return 0;
   }
 
+  getObservableVideoMetricValue(metricName: string, ssrcNum: number): number {
+    const observableVideoMetricSpec = this.observableVideoMetricSpec[metricName];
+    const metricMap = this.getMetricMap(
+      observableVideoMetricSpec.media,
+      observableVideoMetricSpec.dir
+    );
+    const metricSpec = metricMap[observableVideoMetricSpec.source];
+    const transform = metricSpec.transform;
+    const source = metricSpec.source;
+    const streamMetricReport = this.streamMetricReports[ssrcNum];
+    if (
+      streamMetricReport &&
+      observableVideoMetricSpec.source in streamMetricReport.currentMetrics
+    ) {
+      return source
+        ? transform(source, Number(ssrcNum))
```

Comment:
ssrcNum is already a number right ? Why Number cast again ?

Reply:
You are right. We don't need to cast it again. Will change it.